src/components/Home/MenuBar.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import theme from '../../styles/theme';
+
+interface MenuBarProps {
+  theme: typeof theme;
+}
+
+const MenuBar: React.FC<MenuBarProps> = () => {
+  return (
+    <St.MenuBarWrapper>
+      <St.LI>
+        <St.Menu className='home'>홈</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='event'>이벤트</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='movie-talk'>무비톡</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='fast-order'>패스트오더</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='gift-shop'>기프트샵</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='cgv'>@CGV</St.Menu>
+      </St.LI>
+    </St.MenuBarWrapper>
+  );
+};
+
+export default MenuBar;
+
+const St = {
+  MenuBarWrapper: styled.div`
+        background: ${({ theme }) => theme.colors['gradient']};
+        width: 37.5rem;
+        height: 4.2rem;
+        flex-shrink: 0;
+        white-space: nowrap;
+        overflow-x: auto;
+        max-width: 100%;
+        scrollbar-width: none;
+
+        &::-webkit-scrollbar {
+            display: none;
+        }
+    `,
+
+  LI: styled.li`
+        display: inline-block; 
+        list-style-type: none;
+        margin-right: 1.8rem;
+    `,
+
+  Menu: styled.button`
+        color: ${({ theme }) => theme.colors['white']};
+        ${({ theme }) => theme.fonts['body_bold_15']};
+        font-size: 1.5rem;
+        font-style: normal;
+        font-weight: 700;
+        line-height: 100%; 
+        letter-spacing: -0.0075rem;
```

Comment:
P1:  요기! 저거 `${({theme}) => theme.fonts['body_bold_15']}` 요 코드 하나로 그 theme.ts에 미리 정의된 속성들이 다 불러와지는데 그게 font-size부터 letter-spacing에 해당하는거라..! 무슨 말인지 내가 어렵게 말한거 같은데

```suggestion
        ${({ theme }) => theme.fonts['body_bold_15']};
```

요거 하나만 써넣어도 된다!!

Reply:
헐 그렇구나 고마워 바로 수정할게👍👍!!(themes 최고넹...)